monkey/infection_monkey/model/__init__.py

```diff
@@ -5,6 +5,10 @@
 DROPPER_ARG = "dr0pp3r"
 ID_STRING = "M0NK3Y3XPL0ITABLE"
 
+# Dropper target paths
+DROPPER_TARGET_PATH_LINUX = "/tmp/monkey"
+DROPPER_TARGET_PATH_WIN64 = r"C:\Windows\temp\monkey64.exe"
```

Comment:
I think we should move these to the `dropper.py` (if that doesn't cause circular dependencies). Based on this file we can create our own code pattern called "const dumpster".